build.assets/docker/base.dockerfile

```diff
@@ -81,13 +81,21 @@ RUN export DEBIAN_FRONTEND=noninteractive && set -ex \
         automake \
         pkg-config \
         libmnl-dev \
-        make \
-        && apt-get -y autoclean && apt-get -y clean && apt-get autoremove -y \
-        && rm -rf /var/lib/apt/lists/*;
+        make
+
+# Deploy Ubuntu .deb in order to get .socket activated iscsid. Debian's iscsid is not socket activated.
+# Socket activation is needed in order to avoid developing custom service start logic when OpenEBS app is deployed.
+RUN cd /var/lib/dpkg/
```

Comment:
Hm, I'm not sure this has any effect. If you want to run a command inside a specific directory, you can either use a single RUN clause like `RUN cd /var/lib/dpkg && wget ... && apt install ...` or use `WORKDIR` directive. 

In this case I don't think it really matters which directory you run this from, so I think you can just remove this and/or use absolute paths if you want to download it to a specific directory.

Reply:
My two cents - please prefix each RUN statement with `set -ex && ` so the commands are emphasized when the image is built as it's easier to look for build errors that way.
Also, each RUN creates an additional image layer - not sure it's beneficial to split these as they all deal with the same task - pulling and installing the package.

Reply:
And all hardcoded versions should ideally be pulled and configurable from the top-level Makefile and passed as ARGs here.